src/app/zap-templates/templates/app/cluster-structs-header.zapt

```diff
@@ -1,8 +1,18 @@
 {{> header}}
 #pragma once
 
+#include <app/data-model/DecodableList.h>
+#include <app/data-model/List.h>
+#include <app/data-model/Nullable.h>
+#include <app/util/basic-types.h>
```

Comment:
Hmm.  Followup: this should just include DataModelTypes.h, no?  And Percent and company should move into DataModelTypes.h, right?

The only reason we should have basic-types is backwards compat....

Reply:
created #38417 for this